src/ui/assets/AssetsUploadContainer.js

```diff
@@ -1,10 +1,9 @@
 import { connect } from 'react-redux';
 import AssetsUpload from 'ui/assets/AssetsUpload';
-
 import { setVisibleModal, setInfo } from 'state/modal/actions';
 import { UPLOAD_ASSET_MODAL_ID } from 'ui/assets/modals/upload-assets/constants';
 
-export const mapDispatchToProps = dispatch => ({
+export const mapDispatchToProps = (dispatch, { onAssetSelected, buttonVersion, name = '' }) => ({
   onDrop: (assets) => {
```

Comment:
`onDrop` isn't totally appropriate as a name, because it's called when the upload flow is triggered and the resource to upload is selected. What about `onAssetSelected`?